CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Release Toolkit CHANGELOG
+
+---
+
+## Develop
+
+This is our first official release.
+
+### Breaking Changes
+
+_None_
+
+### New Features
+
+_None_
+
+### Bug Fixes
+
+_None_
+
+### Internal Changes
+
+_None_
```

Comment:
Those subsections are just a proposal, but that's what I've been using in most of my OSS projects.
 - If there are any item in "Breaking Changes", this will lead to a major version bump
 - Else, if there are any item in "New Features", this will lead to a minor version bump
 - Else, if there are any item in "Bug Fixes" or "Internal Changes", this will lead to a patch version bump
 - "Internal Changes" is for changes like code refactoring, doc updates and such, which don't fix or change anything in the gem behavior, but are still changes we want to document.

Feel free to suggest another CHANGELOG format, but this one is quite common in OSS world and has worked really well for me.

Reply:
This is excellent, thank you!